<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stenography</class>
 <widget class="QMainWindow" name="Stenography">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stenography</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_encode">
       <attribute name="title">
        <string>Encode</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>511</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Select Image</string>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>471</width>
              <height>31</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="imageToEncodePath"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Select File to Hide</string>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>471</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="fileToEncode"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton_2">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Provide Secret Number</string>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>21</y>
              <width>481</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Secret number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="secretNumberEncode">
               <property name="inputMask">
                <string/>
               </property>
               <property name="maxLength">
                <number>32767</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_11">
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>491</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="encodeButton">
                 <property name="text">
                  <string>Encode</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_decode">
       <attribute name="title">
        <string>Decode</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>21</y>
          <width>511</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Select Image To Decode</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Path</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="imageToDecodePath"/>
              </item>
              <item>
               <widget class="QPushButton" name="browseButton_4">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Provide Secret Number</string>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>491</width>
              <height>51</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Secret number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="secretNumberDecode">
               <property name="inputMask">
                <string/>
               </property>
               <property name="maxLength">
                <number>32767</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>491</width>
              <height>81</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="decodeButton">
                 <property name="text">
                  <string>Decode</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
